package.json

```diff
@@ -5,11 +5,13 @@
   "scripts": {
     "test": "./node_modules/karma/bin/karma start",
     "test:ci:unit": "./node_modules/karma/bin/karma start --browsers PhantomJS --single-run; npm run test:ci:lint",
-    "test:ci:functional": "node ./nightwatch.js -c ./nightwatch.json -e production",
-    "test:ci:lint": "eslint ./src/**/*.js",
     "test:dev:unit": "./node_modules/karma/bin/karma start",
+    "test:ci:functional": "node ./nightwatch.js -c ./nightwatch.json -e production",
     "test:dev:functional": "node ./nightwatch.js -c ./nightwatch.json",
+    "test:ci:lint": "eslint ./src/**/*.js",
     "test:dev:lint": "eslint ./src/scripts/**/*.js",
+    "test:ci:stylelint": "stylelint './src/**/*.scss' --config webpack/.stylelintrc",
+    "test:dev:stylelint": "stylelint './src/**/*.scss' --config webpack/.stylelintrc",
```

Comment:
Good question :+1: 

@mmahalwy i added "ci" and "dev" entries based on existing ""test:dev:lint" & "test:ci:lint". do we really need both separately?